artifactory/commands/dotnet/dotnetcommand.go

```diff
@@ -20,6 +21,12 @@ import (
 const (
 	SourceName        = "JFrogCli"
 	configFilePattern = "jfrog.cli.nuget."
+
+	dotnetTestError = `the command failed with an error.
+Note that JFrog CLI does not restore dependencies during a 'dotnet test' command, so if necessary, run a preceding 'dotnet restore'.
```

Comment:
necessary --> needed
("necessary" has a different meaning than "needed", which is like "if you don't have a choice")